spaces/S000018/properties/P000165.md

```diff
@@ -0,0 +1,8 @@
+---
+space: S000018
+property: P000165
+value: false
+---
+
+Note that the Kolmogorov quotient of $X$, denoted by $Y$, is {S17}
+and {S17|P165}. If $q:X\to Y$ is the Kolmogorov quotient and $H\subseteq Y$ is a countable closed set and $U\supseteq H$ an open set such that there's no open set $V$ with $H\subseteq V\subseteq \text{cl}(V)\subseteq U$, then because $q$ has finite fibers, $H_0 = q^{-1}(H)$ is a countable closed set and $U_0 = q^{-1}(U)$ an open set. If there were open $V_0\subseteq X$ such that $H_0\subseteq V_0\subseteq \text{cl}(V_0)\subseteq U_0$, then $H\subseteq q(V_0)\subseteq \text{cl}(q(V_0)) \subseteq U$ where $V = q(V_0)$ is open, contradiction. So $X$ is not pseudonormal.
```

Comment:
```suggestion
and {S17|P165}. If $q:X\to Y$ is the Kolmogorov quotient map and $H\subseteq Y$ is a countable closed set and $U\supseteq H$ an open set such that there's no open set $V$ with $H\subseteq V\subseteq \text{cl}(V)\subseteq U$, then because $q$ has finite fibers, $H_0 = q^{-1}(H)$ is a countable closed set and $U_0 = q^{-1}(U)$ an open set. If there were open $V_0\subseteq X$ such that $H_0\subseteq V_0\subseteq \text{cl}(V_0)\subseteq U_0$, then $H\subseteq q(V_0)\subseteq \text{cl}(q(V_0)) \subseteq U$ where $V = q(V_0)$ is open, contradiction. So $X$ is not pseudonormal.
```